src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj

```diff
@@ -156,6 +156,7 @@
 			<Choose>
 				<When Condition="'$(Configuration)'=='Debug'">
 					<PropertyGroup>
+						<ProvisioningType>manual</ProvisioningType>
```

Comment:
don't forget to remove this

Reply:
We should not, see previous comment. We had to do this on multiple internal projects so why not have it here.

Reply:
Not having this require that we setup the Apple account in Visual Studio which doesn't work for a lot of people I noticed.

Reply:
Will investigate later today, will wait before merging.

Reply:
let's keep manual because it makes sense for our context, but let's add information in the documentation about it